src/jit/valuenum.cpp

```diff
@@ -155,7 +155,7 @@ T ValueNumStore::EvalOp(VNFunc vnf, T v0, T v1, ValueNum* pExcSet)
             case GT_DIV:
                 if (IsIntZero(v1))
                 {
-                    *pExcSet = VNExcSetSingleton(VNForFunc(TYP_REF, VNF_DivideByZeroExc));
+                    *pExcSet = VNExcSetSingleton(VNForFunc(TYP_REF, VNF_DivideByZeroExc, VNForVoid()));
```

Comment:
Is VNForVoid correct here? The description of VNF_DivideByZeroExc is the divisor value so it seems that here it should the VN of constant 0. Same comment for the next 2 changes bellow.